### ISSUE TITLE
ui5-busyindicator accessibility issues

### ISSUE THREAD

**Issue by ee92** [SAP/ui5-webcomponents#2381]
**Describe the bug**
The ui5-busyindicator is not being read by screenreaders and has no tooltip.

**To reproduce**
1. Using any screenreader (NVDA, JAWS, VoiceOver)
2. go to https://sap.github.io/ui5-webcomponents/playground/components/BusyIndicator/
3. click the 'Fetch List Data' button
4. notice the busy indicator is shown, but nothing read out by the screenreader
5. hover the mouse over any of the 3 busy indicators (Small, Medium, Large) at the top of the page
5. notice there is no tooltip

**Expected behavior**
1. When the busy indicator is shown, the screenreader should read out "Please wait"
2. When hovered, the 3 dots should have a tooltip reading "Please wait"

**Affected components**
ui5-busyindicator

**Organization:**
SuccessFactors

**Priority:**
Medium

Thank you

**Comment by ilhan007** [comment]
Hello @SAP/ui5-webcomponents-topic-rd 

Consider this ACC issue for the BusyIndicator and this PR that tries to address the issues https://github.com/SAP/ui5-webcomponents/pull/2403.

It is up to you if you want to create your own PR/change from scratch or review the existing one https://github.com/SAP/ui5-webcomponents/pull/2403. You can find the issue description and steps to reproduce in the issue description above.

BR,
ilhan

**Comment by ilhan007** [comment]
Hello @ee92 the BusyIndicator's a11y support has been improved baed on your findings in  0.29.13
